### PR TITLE
Try adding group

### PR DESCRIPTION
**Pull request by Hoxbro**
This is meant to declutter the install task by putting all the steps of the actions into their own group, which makes the output collapsible and, therefore, easier to navigate.

I tried to look into if `Run echo "::group::..."` could have a more informative description, but I can see that in the github example, they also have it.

https://octocat.dev/posts/Groups-and-formatting-in-GitHub-Actions
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-grouping-log-lines

### CODE REVIEW

**Comment by Hoxbro** [comment]
Example of (cached) setup: https://github.com/holoviz/holoviews/actions/runs/5353717867/jobs/9710459169

image

**Comment by Hoxbro** [comment]
Example of non-cached output: https://github.com/holoviz/holoviews/actions/runs/5353717867/jobs/9710537164

image

File: install/action.yaml [hunk 1]
```diff
@@ -116,48 +123,65 @@ runs:
...
       run: |
+        echo "::group::Install OpenGL"
         sudo apt-get install libglu1-mesa
         /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x24
         sleep 3
+        echo "::endgroup::"
       shell: bash -el {0}
     - if: inputs.opengl == 'true' && runner.os == 'Linux' && (inputs.cache != 'true' || steps.cache.outputs.cache-hit != 'true')
       run: |
+        echo "::group::Install OpenGL (no cache)"
```
Comment (Hoxbro) [inline install/action.yaml:163]:
```suggestion
        echo "::group::Install mesalib (no cache)"
```

**Comment by maximlt** [comment]
Available at https://github.com/holoviz-dev/holoviz_tasks/releases/tag/v0.1a14

**Comment by Hoxbro** [comment]
I also just discovered it and thought this is what is missing for the install step, as it tends to be very long.

**Pull request merged**